packages/common/src/TimeUtils.ts

```diff
@@ -45,14 +44,3 @@ export async function getFromBlock(web3: Web3): Promise<number> {
     return 0;
   }
 }
-
-/**
- * @notice Estimates the blocks elapsed over a certain number of seconds.
- * @param seconds the number of seconds.
- * @param cushionPercentage the percentage to add to the number as a cushion.
- */
-export async function estimateBlocksElapsed(seconds: number, cushionPercentage = 0.0): Promise<number> {
```

Comment:
Unused outside of `@uma/sdk` and this is redundant with the same function defined there